orion/sites/admin/src/location/locationEdit.js

```diff
@@ -46,22 +109,22 @@ export class LocationEditPage extends React.Component {
             <div id="view-location-edit">
                 <h2>{title}</h2>
                 <h4>Location ID</h4>
-                <input value={this.state.inputLocationId}
-                    onChange={(e) => this.handleChange(e, "inputLocationId")}/>
+                <input inputField={this.state.inputlocId}
```

Comment:
`value` is right here.
```
value={this.state.inputLocId} onChange={(e) => this.handleChange(e, "inputLocId")}
```

Reply:
I guess this is confusing.
value in this part is an HTML thing that is associated with `input`.
It's why in onChange, we can use `value` in `event.target.value`